packages/rn-tester/js/examples/ScrollView/ScrollViewTabViewExample.js

```diff
@@ -0,0 +1,209 @@
+import React, { useCallback, useMemo, useRef } from 'react';
```

Comment:
Could you copy the license from another file, like [this](https://github.com/facebook/react-native/blob/main/packages/rn-tester/js/examples/View/ViewExample.js#L1-L9)? :pray: